client/post-editor/editor-ground-control/history-button.jsx

```diff
@@ -11,37 +11,51 @@ import { flow } from 'lodash';
 /**
  * Internal dependencies
  */
-import { recordTracksEvent } from 'state/analytics/actions';
+import { NESTED_SIDEBAR_NONE, NESTED_SIDEBAR_REVISIONS } from 'state/ui/editor/sidebar/constants';
 import {
-	NESTED_SIDEBAR_NONE,
-	NESTED_SIDEBAR_REVISIONS,
-	NestedSidebarPropType,
-} from 'post-editor/editor-sidebar/constants';
+	closeEditorSidebar,
+	openEditorSidebar,
+	setNestedSidebar,
+} from 'state/ui/editor/sidebar/actions';
+import { getNestedSidebarTarget } from 'state/ui/editor/sidebar/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
 
 class HistoryButton extends PureComponent {
-	toggleShowing = () => {
-		const {
-			isSidebarOpened,
-			nestedSidebar,
-			selectRevision,
-			setNestedSidebar,
-			toggleSidebar,
-		} = this.props;
+	static propTypes = {
+		// passed props
+		// TODO: isSidebarOpened could be connected.
```

Comment:
Maybe add a common tag for the todos so it makes it easier to search + go back to.

Reply:
I've rebased out a few of the comments that weren't really necessary and instead they'll be tracked in the follow up section of the PR description :)